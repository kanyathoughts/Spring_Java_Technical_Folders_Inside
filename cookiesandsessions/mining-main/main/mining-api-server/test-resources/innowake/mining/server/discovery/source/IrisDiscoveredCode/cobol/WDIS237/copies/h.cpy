         01 FIELD-H         PIC X(10).
