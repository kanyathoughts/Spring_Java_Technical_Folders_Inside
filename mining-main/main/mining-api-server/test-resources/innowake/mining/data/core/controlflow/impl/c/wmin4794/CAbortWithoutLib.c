#include <stdio.h>
 
int main(void)
{
   FILE *stream;
 
   if ((stream = fopen("mylib/myfile", "r")) == NULL)
   {
      perror("Could not open data file");
      abort();
   }
   printf("Reach Sometime");
}