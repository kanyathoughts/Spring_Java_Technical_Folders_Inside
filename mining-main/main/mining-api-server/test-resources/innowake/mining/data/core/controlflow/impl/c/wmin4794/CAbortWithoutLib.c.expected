  #4 CFunctionDefinition, Function Name : main, FROM={TestEntryPoint}, TO={#11}
   #11 CExpressionStatement, FROM={#4}, TO={#16}
   #16 CIfElseStatement, Label: if ((stream = fopen("mylib/myfile", "r")) == NULL) { perror("Could not open data file"); abort(); }, FROM={#11}, TO={#30(TRUE), #39(FALSE)}
         #24 CFileAccessStatement
     #30 CExpressionStatement, FROM={#16(TRUE)}, TO={#35}
       #32 CFunctionReference
     #35 CLibraryFunction, FROM={#30}, TO={#39}
       #37 CFunctionReference
   #39 CExpressionStatement, FROM={#16(FALSE), #35}, TO={TestReturnPoint}
     #41 CFunctionReference
